platform/security/servlet/security-servlet-logout/src/main/java/org/codice/ddf/security/servlet/logout/LocalLogoutServlet.java

```diff
@@ -102,4 +106,13 @@ private void deleteJSessionId(HttpServletResponse response) {
     cookie.setComment("EXPIRING COOKIE at " + System.currentTimeMillis());
     response.addCookie(cookie);
   }
+
+  /** Removes OAuth tokens stored for the given session */
+  private void removeTokens(String sessionId) {
+    tokenStorage.delete(sessionId);
```

Comment:
❓ Looks like this is the only place the `delete` method is called. If a user closes their browser rather than logging out, will their token ever be removed?